FirebaseMLModelDownloader/Sources/ModelInfoRetriever.swift

```diff
@@ -35,7 +35,7 @@ extension ModelInfoResponse {
 /// Model info retriever for a model from local user defaults or server.
 class ModelInfoRetriever: NSObject {
   /// Current Firebase app.
-  var app: FirebaseApp
+  private var app: FirebaseApp
```

Comment:
Same note here for a future PR: options is used and would be easier to pass in. Also taking in an `installations` instance as well may make testing easier.

Reply:
Gotcha, updated.